signer/local_cosigner_test.go

```diff
@@ -35,15 +35,16 @@ func TestLocalCosignerGetID(t *testing.T) {
 
 	cosigner := NewLocalCosigner(
 		&RuntimeConfig{},
-		key.ID,
-		*rsaKey,
+		CosignerKeyRSA{
+			ID:     key.ID,
+			RSAKey: *rsaKey,
+		},
```

Comment:
Possibly for a future PR, and a question I've been meaning to ask is why we need the RSA keys.

I found reference to the RSA keys horcrux/docs/signing.md which illuminates some. 

But I think we could expand more explaining why we need them, how to create, and how to manage.